README.md

```diff
@@ -53,4 +53,12 @@ unzip -q eap-microprofile-test-suite-dist.zip
 ```
 Tests output is redirected into `${TEST_FQCN}-output.txt` files in `target/surefire-reports/` directory.
 
-For more details consult with `mp-ts.sh` script
\ No newline at end of file
+For more details consult with `mp-ts.sh` script
+
+## Code format and imports order
```

Comment:
IMHO it would be nice to document formatter deactivation here.

Reply:
Why would we want to promote that use-case ?